### PR TITLE
quick fix for github-pr

### PR DESCRIPTION
**Pull request by babineaum**

### CODE REVIEW

**Comment by babineaum** [comment]
tested 
>(github_env)mbabineau@Matt-Bs-MacBook-Pro:~/dev/jenkins-tools/github-pr$ python github_pr.py list --repo dataxu/dcommand -n 28 --noratelimit
#28 [open]     master <- test-int-ready2                   INT_READY-2
(github_env)mbabineau@Matt-Bs-MacBook-Pro:~/dev/jenkins-tools/github-pr$ python github_pr.py list --repo dataxu/dcommand -n 28
#28 [open]     master <- test-int-ready2                   INT_READY-2
Github Rate Limiting: 4906 remaining of max 5000

**Comment by dx-pbuckley** [comment]
:shipit:

File: github_pr.py [hunk 1]
```diff
@@ -237,7 +237,7 @@ def main():
         github_update_pr(**args)
 
     gh = Github(args['token'])
-    if (('numberonly' in args) and not args['numberonly']) or (('noratelimit' in args) and not args['noratelimit']):
+    if not args['numberonly'] and not args['noratelimit']:
```
Comment (cchristous) [inline github_pr.py]: this change doesn't seem correct. You leave yourself open to exceptions

**Comment by cchristous** [comment]
:-1: What is the point of pull requests if we don't give 5 minutes to review them?

File: github_pr.py (hunk 1 above)
Comment (babineaum) [inline github_pr.py]: this was how it was working before, we need a quick fix. So we went back to what was working. I have another check that will address these issues that i am working on. This was breaking a build.

Comment (cchristous) [inline github_pr.py]: I told you how to fix it before and you said you knew what you wanted. I thought you had tested it. We are now causing unnecessary churn in the master branch

**Pull request merged**